### DISCUSSION TITLE
MissionControl 0.9.3

### DISCUSSION THREAD

**Discussion by ndeadly** [ndeadly/MissionControl#685] [General]
<p align="left">
<img alt="GitHub Releases" src="https://img.shields.io/github/downloads/ndeadly/MissionControl/v0.9.3/total">
<a href="https://discord.gg/gegfNZ5Ucz"><img alt="Discord Server" src="https://discordapp.com/api/guilds/905069757361971303/widget.png?style=shield"/></a>
</p>

This is the twentieth official release of Mission Control.

This release adds support for firmware version 16.1.0, along with some new controllers, settings, and bugfixes. See below for the full changelog.

New users landing on this release page should first check out the readme on the [main project page](https://github.com/ndeadly/MissionControl#readme) for the official project documentation. There you can find installation and usage instructions along with an FAQ section that will answer most of your questions.

_PS: there is some ongoing testing of preliminary Bluetooth LE support happening in my discord server (see #testing). If you own any of the controllers listed [here](https://github.com/ndeadly/MissionControl/issues?q=is%3Aissue+is%3Aopen+label%3A%22BLE+controller%22) or any LE controllers that are not on the list, we'd like to hear from you! (though I don't need any additional input on Xbox and Stadia controllers for now). Please note that these are alpha builds for the purpose of gathering controller information and furthering my understanding of how BLE works on the console. They are **_NOT_** intended for general use. There _will_ be bugs and issues. There are no ETAs. I will not be providing any support - if you can't follow the usage instructions, then this is not for you._

### Changelog:
- Added support for Steelseries Stratus XL controller
- Added support for 8BitDo Ultimate 2.4G Wireless Controller
- Added support for Betop 2585N2 controller
- Added .ini config option for setting the analog trigger activation threshold
- Added new "hybrid" mode to .ini settings for Dualshock 3 player indicator LEDs to display the Switch LED pattern, but reversed to align with the numbers printed on the controller
- Changed behaviour of Dualshock 4/Dualsense touchpad so that the outer edges are mapped to +/- in order to prevent accidental screenshots being taken when pressing those buttons
- Dualsense mute button is now additionally mapped as capture button
- Updated list of official controller names to include a new name introduced in HOS 15.0.0. This fixes an issue where some newer PowerA licensed Switch controllers don't register inputs with Mission Control running.
- Fixed a bug that could cause the console to spam commands when trying to set the player indicator LEDs to flash

<hr /><em>This discussion was created from the release <a href='https://github.com/ndeadly/MissionControl/releases/tag/v0.9.3'>MissionControl 0.9.3</a>.</em>

**Comment by 916714315** [comment]
Do you support the latest one Xbox Series X？

**Reply by ndeadly** [reply]
No, not currently. It uses Bluetooth LE, a different Bluetooth protocol. Work to support it is ongoing.

**Reply by 916714315** [reply]
Looking forward to your update, thank you very much, I will always support you!

**Comment by tuanzi11** [comment]
这个版本插件手柄连上   ，主机死机！

**Reply by ndeadly** [reply]
GitHub is an international website, please communicate in English. If this is not possible for you, please translate with a service like [deepl](https://www.deepl.com/en/translator-mobile)

**Reply by tuanzi11** [reply]
The new version of the plug-in will crash when connecting to the host

**Reply by ndeadly** [reply]
Which controller are you trying to connect?

This is likely a problem with your setup, everything works fine for me. Can you show me your `/atmosphere/contents` directory?

**Reply by tuanzi11** [reply]
![image](https://github.com/ndeadly/MissionControl/assets/92656905/fcf6acd7-6baa-4b78-97b0-f3b1f4131865)
My friend's xbox controller won't connect!

**Reply by tuanzi11** [reply]
![image](https://github.com/ndeadly/MissionControl/assets/92656905/c60bf87c-0016-4e4f-97f1-f590a29fd3b7)

**Reply by ndeadly** [reply]
What are the two that are covered up? I want to see everything else you have installed, not just that Mission Control is present.

> My friend's xbox controller won't connect!

Is this a separate issue, or is this what you are describing? Series X|S controller and Xbox One controllers with firmware version 5.x.x.x and above are not currently supported. In the case of Xbox One controllers they can be downgraded to a compatible firmware.

**Reply by Masamune3210** [reply]
1013 is the mypage applet, no idea what that would need a mod for
000d is dmnt, which isn't even normally present on retail
Other than Tesla and mc, the top one is apparently PointerSearcher

**Reply by ndeadly** [reply]
Everything seems relatively harmless. If it's an Xbox controller, I have heard the odd report of them causing an occasional crash during pairing. No idea why, nobody has every sent me an error report. But the solution seems to have been to clean the bluetooth pairing database or delete `/config/MissionControl/controllers`

**Reply by tuanzi11** [reply]
Connecting the switchpro joystick died! The interface is just stuck!

**Reply by ndeadly** [reply]
Somebody reported the same thing to me earlier. It was a problem with their custom theme (though I don't think you have the same files). I would start by removing everything in `/atmosphere/contents` besides Mission Control and go from there.

**Reply by Masamune3210** [reply]
A theme caused the system to crash when pairing a controller?

**Reply by tuanzi11** [reply]
> A theme caused the system to crash when pairing a controller?

no theme installed

**Reply by ndeadly** [reply]
> A theme caused the system to crash when pairing a controller?

Yeah, well, not a crash but hanging the system. It was applying mods to the playerSelect applet (Change Grip/Order screen) that I assume were outdated and freezing up when the user tried to pair an official controller.

> no theme installed

I don't know where title `0100000000001013` in your contents folder came from, but titles in the `0100000000001XXX` range are applets typically targeted by themes. I would just delete everything from the `/atmosphere/contents` directory and reinstall Mission Control, reboot, and confirm whether the issue is still present. If it is I'll investigate further, but I suspect it's probably caused by something else.

**Reply by Masamune3210** [reply]
1013 would be a mod for the player screen, i.e. this
![image](https://github.com/ndeadly/MissionControl/assets/1053504/623a4d9f-2f0e-4fd1-9ce7-0f33205f1369)

**Reply by tuanzi11** [reply]
I have summed up the problem. After decompressing this package, you must delete the controller plug-in link to the pro controller so that the machine does not crash.

**Reply by ndeadly** [reply]
@tuanzi11 if you have a crash report please post it

**Reply by tuanzi11** [reply]
No bugs to report!  It's dead.

**Reply by ndeadly** [reply]
If it was crashing earlier it should have generated crash reports on your sd card. I would be curious to know what the actual error was, because deleting paired controllers should only be required in exceptional cases.

**Reply by a0930530** [reply]
Crash when connecting to WiiU PRO

**Comment by a0930530** [comment]
Unable to pair, PS4 and WiiU PRO

**Reply by ndeadly** [reply]
See [FAQ ](https://github.com/ndeadly/MissionControl#frequently-asked-questions)section titled `I installed Mission Control but it doesn't do anything. Help!`

**Reply by a0930530** [reply]
Has anyone else been able to use the latest version of PS4 and WIIU PRO

**Reply by a0930530** [reply]
16.1.0+atmosphere1.5.5+MissionControl 0.9.3
Unable to pair, PS4 and WiiU PRO

**Reply by ndeadly** [reply]
Dude.

**Reply by ndeadly** [reply]
I own both controllers and use them for testing. They work. Go and read the FAQ section I linked. It has all the troubleshooting steps I would otherwise tell you to do. If you've read through them, then walk me through what you did and what happened. 99% of the time "unable to pair" comes down to user error or a bad install. I can't possibly know the cause of the issue if you don't tell me what you did.

**Reply by a0930530** [reply]
Is your version atmosphere1.5.5 ???????

**Reply by ndeadly** [reply]
Of course. I'm the developer. I can't put out an update supporting 16.1.0 without running both that and the latest atmosphere. Please just read the FAQ man...

**Comment by a0930530** [comment]


**Reply by ndeadly** [reply]
Don't double post. See my previous reply https://github.com/ndeadly/MissionControl/discussions/685#discussioncomment-6820714
